File: k9mail/src/main/java/com/fsck/k9/notification/NotificationGroupKeys.java
package com.fsck.k9.notification;


import com.fsck.k9.Account;


class NotificationGroupKeys {
    private static final String NOTIFICATION_GROUP_KEY_PREFIX = "newMailNotifications-";
    
    
    public static String getGroupKey(Account account) {
        return NOTIFICATION_GROUP_KEY_PREFIX + account.getAccountNumber();
    }
}

File: app/core/src/main/java/com/fsck/k9/notification/NotificationGroupKeys.kt
package com.fsck.k9.notification;


import com.fsck.k9.Account;


public class NotificationGroupKeys {
    private static final String NOTIFICATION_GROUP_KEY_PREFIX = "newMailNotifications-";
    
    
    public static String getGroupKey(Account account) {
        return NOTIFICATION_GROUP_KEY_PREFIX + account.getAccountNumber();
    }
}
